source/usage-examples/deleteMany.txt

```diff
@@ -3,3 +3,52 @@ Delete Multiple Documents
 =========================
 
 .. default-domain:: mongodb
+
+You can delete multiple documents in a collection by using the
+``DeleteMany()`` method. 
+
+The following example specifies a query filter that matches documents in
+the ``movies`` collection where the ``runtime`` field is greater than
+*800*, and deletes all the matching documents.
```

Comment:
Issue: Something about the wording here is off, I don't think there should be a comma before "and deletes...". It's somewhat unclear what is doing the deleting-- the example of the query filter. Here's the wording from the updateMany page: 

"The following example passes a query filter and an update parameter to the UpdateMany() method, which matches documents in the listingsAndReviews collection where the value of the address.market field is "Sydney" and multiplies the price field in the matching documents by a factor of 1.15"

Reply:
Good catch! I'll update it to a similar wording as yours.